src/pages/_proxied-dot-io/vault/home/index.tsx

```diff
@@ -0,0 +1,173 @@
+import * as React from 'react'
```

Comment:
nit: we could move this to a view, and remove the `/home` route altogether. Currently we've got a redirect in place from `/home` to `/`

(This can also be a follow-up refactor)

Reply:
Went ahead and did this :D